src/Assets/Bugsnag/Scripts/BugsnagSettingsObject.cs

```diff
@@ -40,6 +40,12 @@ public class BugsnagSettingsObject : ScriptableObject
         public double SecondsPerUniqueLog = 5;
         public List<TelemetryType> Telemetry = new List<TelemetryType> { TelemetryType.InternalErrors };
         public int VersionCode;
+
+        public SwitchCacheType SwitchCacheType = SwitchCacheType.R;
+        public string SwitchCacheMountName = "BugsnagCache";
+        public int SwitchCacheIndex = 0;
+        public int SwitchMaxCacheSize = 10485760;
```

Comment:
Suggest we rename this for consistency - perhaps that requires the underlying config option to change too? But feels worth it to save confusion.

```suggestion
        public int SwitchCacheMaxSize = 10485760;
```

Reply:
Done